src/Umbraco.Core/Migrations/Upgrade/V_8_7_0/StackedContentToBlockList.cs

```diff
@@ -141,11 +170,11 @@ private void UpdateDataType(DataTypeDto dataType)
         private class BlockListConfiguration
         {
 
-            [ConfigurationField("blocks", "Available Blocks", "views/propertyeditors/blocklist/prevalue/blocklist.elementtypepicker.html", Description = "Define the available blocks.")]
+            [JsonProperty("blocks")]
```

Comment:
Not sure why these changes are in this PR? Also i might just not be in the loop on this one but why are we removing the [ConfigurationField] attributes for things that are configuration fields?

Reply:
oooooh .... haha, these are copied locally here as private classes specific to the migration, ok i gotcha